ClubService.Domain/Event/Member/MemberAccountLimitExceededEvent.cs

```diff
@@ -1,10 +1,5 @@
 namespace ClubService.Domain.Event.Member;
 
-public class MemberAccountLimitExceededEvent(
-    Guid eventId,
-    Guid entityId,
-    EventType eventType,
-    EntityType entityType)
-    : Event(eventId, entityId, eventType, entityType)
+public class MemberAccountLimitExceededEvent : IMemberDomainEvent
```

Comment:
This event is of type ITennisClubEvent because we need to know for which Tennis Club it occurs.

Reply:
Also it needs to be moved to the TennisClub directory.